docs/getting-started/developers/index.md

```diff
@@ -56,23 +56,42 @@ When setting up Warp in your project, you can choose to create an `uno.config` f
 
 - **Alternative 1: Add a uno.config file**
 
-Create a `uno.config.[js,ts,mjs,mts]` file with the following content. This file will configure UnoCSS with our Warp preset, including a safelist of component classes, which will add styling to Warp components. See all configuration options for `presetWarp` in the [Warp CSS docs](https://warp-ds.github.io/css-docs/plugin-api).
+Create a `uno.config.[js,ts,mjs,mts]` file with the following content. This file will configure UnoCSS with our Warp preset. See all configuration options for `presetWarp` in the [Warp CSS docs](https://warp-ds.github.io/css-docs/plugin-api).
 
 > uno.config.js
 
 ```js
 import { defineConfig } from 'unocss';
 import { presetWarp } from '@warp-ds/uno';
-import { classes } from '@warp-ds/component-classes/classes';
 
 export default defineConfig({
   presets: [presetWarp()],
-  safelist: classes,
 });
 ```
 
 By default, UnoCSS will automatically look in the root directory of your project for `uno.config.[js,ts,mjs,mts]` or `unocss.config.[js,ts,mjs,mts]`.
 
+#### Components
+
+If you are using Warp components there are two additional steps to do.
+
+1. You'll need to add `https://assets.finn.no/pkg/@warp-ds/css/1.0.0-alpha.10/components.css` in the head of your app or in the template if you are using such. This stylesheet is needed to add styling to Warp components.
```

Comment:
Why do we suggest using `1.0.0-alpha.10` precisely? I see that this file is available from version `1.0.0-alpha.9` and we're currently at `1.0.0-alpha.22`. I suppose it's for compatibility with some version of `@warp-ds/uno` but should we mention it here, too so it's clear why users need to update it manually?

Reply:
Also, if it's a matter of compatibility, we suggest installing `@warp-ds/uno@alpha` at the beginning of these docs, which will install the latest alpha version, and that might not necessarily be compatible with `1.0.0-alpha.10` of component-classes, if I'm understanding things correctly.

Reply:
I asked the question about version here https://github.com/warp-ds/tech-docs/pull/82#discussion_r1250441648. I just added something to have a discussion

Reply:
Ah, I didn't see that, sorry!

Reply:
I really think that the latest published version of uno should be compatible with the latest published version of css repo parts. Version `1.0.0-alpha.10` is just an example.

We need to align how to do here. Maybe we will have an alias here as we did before.

Reply:
> Version 1.0.0-alpha.10 is just an example.

If I looked at the docs, I'd think this is the exact version I should use 😄

> Maybe we will have an alias here as we did before. 

It could be we'd need to publish version `1.0.0` first to give an impression that no breaking changes are expected on v1. Until then maybe we could write `https://assets.finn.no/pkg/@warp-ds/css/<version>/components.css` here in the docs and link to a README on [warp-ds/drive](https://github.com/warp-ds/drive) where we show which versions of `@warp-ds/css` and `warp-ds/uno` are compatible, e.g. in a form of a table?